frontend/src/App.js

```diff
@@ -84,6 +82,17 @@ const App = observer(
       });
     };
 
+    adminLink = () => {
+      // Need to figure out the right url that works in the local dev environment
+      // which does this proxy thing
+      let here = window.location;
+      let url =
+        here.protocol + "//" + here.hostname + ":" + here.port + "/admin";
+      // Replace proxy webpack host with django host in local dev environment
+      url = url.replace("http://localhost:3000/", "http://localhost:8000/");
```

Comment:
This kind of stuff is kind of gross, but there's no other way to deal with the proxy in the local dev environment.